pkgs/development/tools/unused/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPackages_1_45, cmake }:
+rustPackages_1_45.rustPlatform.buildRustPackage rec {
+  pname = "unused";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "unused-code";
+    repo = pname;
+    rev = version;
+    sha256 = "0ppfcmav3l53bf9lq3wqdbfblq3vsskd4hhilkp2d44767ilmiwv";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  cargoSha256 = "1g7vdizmxc1drq5jwiqgbg6zlk3cwmvwd0kcg9n3rhjxdq1jryaw";
+
+  meta = with stdenv.lib; {
+    description = "A tool to identify potentially unused code";
+    homepage = "https://unused.codes";
+    license = licenses.mit;
+    maintainers = [ ];
```

Comment:
Consider adding yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix if you want to maintain this package